app/src/main/res/values/strings.xml

```diff
@@ -884,29 +884,29 @@
     <string name="network_log_app_name_unknown">Unknown</string>
     <string name="network_log_app_name_unnamed">Unnamed (%1$s)</string>
 
-    <string name="dns_crypt_connection_failure">Error connecting to DNSCrypt server.</string>
-    <string name="dns_proxy_connection_failure">Error connecting to DNS Proxy server.</string>
+    <string name="dns_crypt_connection_failure">Could not connect to DNSCrypt server.</string>
+    <string name="dns_proxy_connection_failure">Could not connect to DNS proxy server.</string>
```

Comment:
uppercase `P` preferred

Reply:
The "proxy" isn't intrinsic to DNS. I think either "DNS-proxy server" or "DNS proxy-server".